web-app/src/InvoiceDocumentation/ubl2.1/rules/index.js

```diff
@@ -0,0 +1,28 @@
+import {useEffect} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+import {Route, Switch} from 'react-router-dom'
+import Overview from './Overview'
+import Rule from './Rule'
+import {isUblRulesDocsLoadedSelector} from '../../../cache/documentation/state'
+import {getUblRulesDocs} from '../../../cache/documentation/actions'
+
+export default ({match}) => {
+  const isLoaded = useSelector(isUblRulesDocsLoadedSelector)
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    if (!isLoaded) {
+      dispatch(getUblRulesDocs())
+    }
+  }, [dispatch, isLoaded])
+
+  // Data is loading
+  if (!isLoaded) return null
+
+  return (
+    <Switch>
+      <Route exact path={match.url} component={Overview} />
+      <Route component={Rule} />
```

Comment:
let's change it to
```
<Route path={`${match.url}/:id`} component={Rule}/>
<Route component={Overview}>
```
then you don't have to parse id from URL but it will be accessible in `match.url.params` in `Rule` component